vic/drivers/shared_image/src/vic_init.c

```diff
@@ -986,8 +986,7 @@ vic_init(void)
                 }
                 sum += soil_con[i].AreaFract[j];
             }
-            // TBD: Need better check for equal to 1.
-            if (sum != 1.) {
+            if (!assert_close_double(sum, 1.0, 0., 0.001)) {
```

Comment:
Generally I prefer to have the work done by the user since that is the most explicit. They should only need to do this once and it seems to me that that would be an easy calculation in some version of `nco`, `cdo` or `xarray`. `0.001` would seem too large to me to take care of behind the scenes. The one thing is that our documentation needs to be very clear on what we expect from the user, so if we print an error message, then our documentation needs to be explicit on how to fix it or prevent it.